### PR TITLE
Reworked controllers.py variable names to match Ratik's TypeScript

### PR DESCRIPTION
**Pull request by dimitar-j**

### CODE REVIEW

File: systems/controllers.py [hunk 1]
```diff
@@ -91,11 +93,15 @@ class ClimateController:
...
 
     Attributes:
-        weathertemperature (int): Defaults to 0.
+        outsidetemperature (int): Defaults to 0.
+        insidetemperature (int): Defaults to 0.
         fanpower (int): Defaults to 0.
+        temperaturesetting (int): Defaults to 0.
     """
-    weathertemperature: int = 0
+    outsidetemperature: int = 0
```
Comment (RatikKapoor) [inline systems/controllers.py:101]: @DhyeyLalseta @rjb75, would it be better to camelCase these variables or is it fine all lower case?

Comment (dimitar-j) [reply systems/controllers.py:101]: Oh yeah I was thinking that too, I just decided to keep it all lowercase to match the other variables

Comment (DhyeyLalseta) [reply systems/controllers.py:101]: completely up to you all, my personal opinion i like all lowercase a bit better but no strong feelings towards any

Comment (RatikKapoor) [reply systems/controllers.py:101]: Hmm, yeah I think camelCase would be better. It's more readable and the exact same variable names will need to be used in the UI as well
image

Comment (RatikKapoor) [inline systems/controllers.py:101]: Switching to camelCase for readability!

**Review by RatikKapoor — CHANGES_REQUESTED** [review]

**Review by RatikKapoor — APPROVED** [review]
🚀

**Review by DhyeyLalseta — APPROVED** [review]

**Pull request merged**